src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -105,6 +114,20 @@ public override string SubProtocol
                 websocketControl.SupportedProtocols.Add(subProtocol);
             }
 
+            if (MessageWebSocketClientCertificateSupported)
```

Comment:
We should not throw exceptions for this kind of platform difference.  See prior HttpClient UWP PRs. In general, we need to avoid throwing PNSE because it breaks developers.